frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -152,11 +157,25 @@ function Pages(): JSX.Element {
                         identifier={Scene.SavedInsights}
                         to={urls.savedInsights()}
                         sideAction={{
-                            icon: <IconPlus />,
-                            to: urls.insightNew(),
-                            tooltip: 'New insight',
                             identifier: Scene.Insight,
-                            onClick: hideSideBarMobile,
+                            tooltip: 'Choose an insight type to create',
+                            onClick: () => setIsNewInsightMenuShown((state) => !state),
+                            dropdown: {
+                                visible: isNewInsightMenuShown,
+                                onClickOutside: () => setIsNewInsightMenuShown(false),
+                                onClickInside: hideSideBarMobile,
+                                overlay: (
+                                    <div className="SideBar__side-actions" data-attr="sidebar-new-insights">
+                                        <h5>New Insight</h5>
```

Comment:
"New Insight" sounds really dry, how about something describing the _action_ being undertaken
```suggestion
                                        <h5>Creating a new insight</h5>
```